### ISSUE TITLE
RearViewController's position can be static?

### ISSUE THREAD

**Issue by adarshsripathyrao** [John-Lluch/SWRevealViewController#594]
Hi, First of all thank you very much for creating this library. Have made my life much more simpler. I have used it in most of my applications. 
I have a question on how to keep the position of the rearviewcontroller static and not pan it across when the front view controller is getting panned? Please suggest.

**Comment by iDevelopper** [comment]
rearViewController.rearViewRevealOverdraw = 0

**Comment by adarshsripathyrao** [comment]
Tried it and its not working.

**Comment by iDevelopper** [comment]
Sorry!

```
    rearViewRevealDisplacement = 0
```

**Comment by adarshsripathyrao** [comment]
Still Not working.

**Comment by iDevelopper** [comment]
Then perhaps I did not understand the question!

**Comment by iDevelopper** [comment]
What about this?

[SWStaticRear.zip](https://github.com/John-Lluch/SWRevealViewController/files/318533/SWStaticRear.zip)

**Comment by adarshsripathyrao** [comment]
No that is not what I wanted. I'll re-iterate my question, I have a menu bar which is a FrontViewController. Now when I am in any other view controller I must be able to swipe in the menu bar via pan gesture. This is easily achievable. But I also want the menu bar to come on top of the rear view controller as an overlay. Or in other words, I don't want my rear view to move along with Front View Controller. How do I make the menu bar come on top of other view controllers as an overlay using SWRevealViewController?

**Comment by iDevelopper** [comment]
I can't understand your requirement as the front view move but not the rear! Perhaps you mean you want the menu on top of front? In this case you use front for menu and right for controllers. If it is I have somme samples.

**Comment by adarshsripathyrao** [comment]
Yes, I basically want my menu bar to come as an overlay. And yes please share any sample code. It would be of much help. Thanks

**Comment by iDevelopper** [comment]
[SWTopMenu.zip](https://github.com/John-Lluch/SWRevealViewController/files/320083/SWTopMenu.zip)
[SampleProj.zip](https://github.com/John-Lluch/SWRevealViewController/files/320089/SampleProj.zip)
[SWFrontMenu.zip](https://github.com/John-Lluch/SWRevealViewController/files/320095/SWFrontMenu.zip)
[SWMenuOnTop.zip](https://github.com/John-Lluch/SWRevealViewController/files/320097/SWMenuOnTop.zip)

**Comment by adarshsripathyrao** [comment]
Hi, sorry for the late response, this works well. Thanks!! One more query, Can this be done if I am having two menu bars opening on pan right and pan left. And both should come as an overlay.?
